src/interactions/map_arcade/collectCoins.interactions.js

```diff
@@ -0,0 +1,10 @@
+export const collectCoins = (player, k, map) => {
```

Comment:
Thoughts on a new directory in interactions called ```utils``` and to put ```collectCoins``` in there? This feels more a like a util since we have it in each map_xxx directory.

Can probably move ```handler.interactions.js``` in the new utils directory too

Reply:
I have implemented this into the code just now